ubuntu_18.04-dev.dockerfile

```diff
@@ -2,17 +2,27 @@ FROM ubuntu:18.04
 
 ENV HOME /root
 
+# install apt dependencies
 RUN apt-get -y  update
-RUN apt-get install -y  \
-    software-properties-common wget
-
-# pyne specific dependencies (excluding python libraries)
-RUN apt-get install -y build-essential git cmake vim emacs gfortran libblas-dev \
-                       python-pip liblapack-dev libeigen3-dev libhdf5-dev autoconf libtool
+RUN apt-get install -y software-properties-common \
+                       python3-pip \
+                       wget \
```

Comment:
I don't think we need this, do we?

Reply:
Not entirely sure, but I think `wget` might be used inside PyNE for downloading some things? I will double check.

Reply:
Yes, that sounds right